Core/src/main/java/com/plotsquared/core/command/Visit.java

```diff
@@ -155,7 +155,7 @@ private void visit(@NotNull final PlotPlayer player, @NotNull final PlotQuery qu
         switch (args.length) {
             // /p v [...] [...] <page>
             case 3:
-                if (!MathMan.isInteger(args[1])) {
+                if (!MathMan.isInteger(args[2])) {
```

Comment:
Is this then `/plot visit <user> <area> <number>`?

Reply:
Yes, seems to be the only remaining part with 3 arguments. So the usage should be `/plot visit <player>|<alias>|<plot> [area] [#]` I guess?

Reply:
it's a bit messier than that, `/plot visit <player>|<alias>|<plot> [area]|[#] [#]` lol